drivers/modem/gsm_ppp.c

```diff
@@ -557,7 +559,7 @@ static void mux_setup(struct k_work *work)
 {
 	struct gsm_modem *gsm = CONTAINER_OF(work, struct gsm_modem,
 					     gsm_configure_work);
-	const struct device *uart = device_get_binding(CONFIG_MODEM_GSM_UART_NAME);
+	const struct device *uart = device_get_binding(GSM_UART_DEV_NAME);
```

Comment:
Please use DEVICE_DT_GET instead of device_get_binding.

Reply:
@mbolivar May I ask why? This would make this line less consistent with functions (`modem_iface_uart_init*`) that use `GSM_UART_DEV_NAME`

Reply:
There is no reason to use device_get_binding if you have a devicetree node. DEVICE_DT_GET fails at build time if the device does not exist and is faster at runtime if it does.

Reply:
Should be fixed now